config/locales/devise.hi.yml

```diff
@@ -0,0 +1,60 @@
+hi:
+  devise:
+    confirmations:
+      confirmed: 'आपके ईमेल की सफलतापूर्वक पुष्टि हो गई है।'
+      send_instructions: 'आपको कुछ ही मिनटों में अपने ईमेल की पुष्टि करने के निर्देश के साथ एक ईमेल प्राप्त होगा।'
+      send_paranoid_instructions: 'यदि आपका ईमेल हमारे डेटाबेस में मौजूद है, तो आप कुछ ही मिनटों में अपने ईमेल पते की पुष्टि करने के निर्देश के साथ एक ईमेल प्राप्त करेंगे।'
+    failure:
+      already_authenticated: 'आप पहले ही साइन इन कर चुके हैं।'
+      inactive: 'आपका खाता अभी तक सक्रिय नहीं किया गया है।'
+      invalid: 'अमान्य %{authentication_keys} या पासवर्ड।'
+      locked: 'आपका खाता लॉक है।'
+      last_attempt: 'आपके खाते के लॉक होने से पहले आपके पास एक और प्रयास है।'
+      not_found_in_database: 'अमान्य %{authentication_keys} या पासवर्ड।'
+      timeout: 'आपका सत्र समाप्त हो गया। जारी रखने के लिए कृपया पुन: प्रवेश करें।'
+      unauthenticated: 'जारी रखने से पहले आपको साइन इन या साइन अप करने की आवश्यकता है।'
+      unconfirmed: 'जारी रखने से पहले आपको अपने ईमेल पते की पुष्टि करनी होगी।'
+    mailer:
+      confirmation_instructions:
+        subject: 'पुष्टि निर्देश'
+      reset_password_instructions:
+        subject: 'पासवर्ड रीसेट करने के निर्देश'
+      unlock_instructions:
+        subject: 'अनलॉक करने के निर्देश'
+      password_change:
+        subject: 'पासवर्ड बदला गया'
+    omniauth_callbacks:
+      failure: 'आपको %{kind} से प्रमाणित नहीं किया जा सकता क्योंकि %{reason}।'
+      success: '%{reason} खाते से सफलतापूर्वक प्रमाणित किया गया।'
+    passwords:
+      no_token: 'आप पासवर्ड रीसेट ईमेल से आए बिना इस पृष्ठ तक नहीं पहुंच सकते। यदि आप एक पासवर्ड रीसेट ईमेल से आते हैं, तो कृपया सुनिश्चित करें कि आपने पूर्ण URL प्रदान किया है।'
```

Comment:
I see other places use "लिंक" so maybe "URL" can be changed to "लिंक" here?

Reply:
You're right. I'd change this.